.github/workflows/ci.yaml

```diff
@@ -256,11 +266,24 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          file: .github/containers/test-installation/Dockerfile
+          file: .github/containers/test-installation/python-${{ matrix.python }}.Dockerfile
           platforms: linux/amd64,linux/arm64
           tags: localhost/test-installation
           push: false
 
+  # This job runs if all the `test-installation` matrix jobs ran and succeeded.
+  # It is only used to have a single job that we can require in branch
+  # protection rules, so we don't have to update the protection rules each time
+  # we add or remove a job from the matrix.
+  test-installation-all:
+    # The job name should match the name of the `test-installation` job.
+    name: Test package installation in different architectures
+    needs: ["test-installation"]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Return true
+        run: "true"
+
```

Comment:
There is an updated fixed version of these "merging" jobs so merge queues can really check them. You can have a look at repo-config.

Reply:
oh yes, I now remember about this as I reviewed it :/
I'll apply those changes here and probable other similar jobs if needed

Reply:
Addressed